frontend/src/utils/getRequest.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+
+const MEDICATION_LIST_ENDPOINT = "https://jsonplaceholder.typicode.com/todos";
+
+const sendGetRequest = (body) => {
```

Comment:
I would suggest to create new folder "api" to store code related to api requests

Reply:
Also I would suggest to rename function to sth more verbose, e.x: getMedicationsList